### DISCUSSION TITLE
Not found results from queries

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5738] [Feature Requests]
> I feel like this is a luxury request and is a long-shot in terms of how much would have to be built to support this - but when using the ‘in’ option on a query followed by a comma-separated list, it would be handy for the query to identify or report which requested items weren’t returned in the query.

>Not exactly… I’d like it to identify items that are missing completely from the database. For example, if I intended to catalog items with catalog numbers 143959 through 146323, I would like to query that range using “in,” supply it with the list of expected numbers, and see if I missed any in the results. something like “Not found: 144835, 14523” would be helpful. As is, I can just export and open the results in Excel to inspect for missing items, hence this would be a ‘luxury’ feature.

**Requested By:** Paul at FWRI ([Speciforum](https://discourse.specifysoftware.org/t/not-found-results-from-queries/1028))

**Comment by specifysoftware** [comment]
This issue has been mentioned on **Specify Community Forum**. There might be relevant details there:

https://discourse.specifysoftware.org/t/not-found-results-from-queries/1028/4

**Comment by grantfitzsimmons** [comment]
**Requested By:** Nate Shoobs at Ohio State

> Just saw this post for the first time, and wanted to add Ohio State to the list of institutions that want this feature. It would be handy, as otherwise the only way to know what you’re missing is by exporting a .csv of the query and checking the results against the query using a formula/script.
